include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -426,26 +426,31 @@ def InvokeOp : CalyxOp<"invoke", [
     call, which invokes a given component.
 
     The 'callee' attribute is the name of the component,
-    the 'ports' attribute The ports attribute specifies 
-    the input port of the component when it is invoked, 
-    and the 'inputs' attribute specifies the assignment on 
-    the corresponding port.
+    the 'ports' attribute specifies the input port of the component when it is invoked,
+    the 'inputs' attribute specifies the assignment on the corresponding port,
+    and the 'refCellsMap' attribute maps the reference cells in the `callee` with
+    the original cells in the caller.
 
     ```mlir
+      calyx.component @identity {
+        %mem.addr0, %mem.clk, ... = calyx.seq_mem @mem <[1] x 32> [1] {external = false}
+      }
       %id.in, %id.out, ... = calyx.instance @id of @identity : i32, i32, ...
       %r.in, ... = calyx.register @r : i32, ...
+      %mem_1.addr0, %mem_1.clk, ... = calyx.seq_mem @mem_1 <[1] x 32> [1] {external = true}
       ...
       calyx.control {
         calyx.seq {
-          calyx.invoke @id(%id.in = %c1_10, %r.in = %id.out) -> (i32, i32)
+          calyx.invoke @id[mem = mem_1](%id.in = %c1_10, %r.in = %id.out) -> (i32, i32)
```

Comment:
Can you update the native Calyx emitter and add a test please?

Reply:
Hey @jiahanxie353 do you mind resolving these comments if you addressed the issues, or following up if you want some clarification / disagree with something? Thank you!

Reply:
Oops sorry I didn't check in in the first place!

Thanks for reviewing and reminding me @cgyurgyik !
Hmm, I thought I have emitter check [here](https://github.com/llvm/circt/blob/9af79669551d9b8bf09f258c5256df4d4a6494f9/test/Dialect/Calyx/emit-invoke-with-ref.mlir#L61). Is this not we are looking for?

Reply:
Right, so if you addressed the comment, can you mark this conversation as resolved?

Reply:
It just makes it easier to determine whether you've addressed it or not.